dojo/unittests/tools/test_checkov_parser.py

```diff
@@ -22,3 +22,10 @@ def test_parse_file_with_multiple_vuln_has_multiple_findings(self):
         parser = CheckovParser()
         findings = parser.get_findings(testfile, Test())
         self.assertTrue(len(findings) > 2)
+
+    def test_parse_file_with_multiple_check_type_has_multiple_check_type(self):
+        testfile = open("dojo/unittests/scans/checkov/checkov-multiple-check_type.json")
+        parser = CheckovParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertTrue('Check Type: terraform' in findings[0].description)
```

Comment:
could you at least add a check on number of findings?
```suggestion
        self.assertTrue(len(findings) > ???)
        self.assertTrue('Check Type: terraform' in findings[0].description)
```

Reply:
Done at https://github.com/DefectDojo/django-DefectDojo/pull/4705/commits/1272cae9fdd3423317210e93266b2f84bc1ed2f9